tests/robustness/snapmeta/kopia_persister_light.go

```diff
@@ -45,6 +46,22 @@ func (kpl *KopiaPersisterLight) ConnectOrCreateRepo(repoPath string) error {
 	return kpl.kc.CreateOrConnectRepo(context.Background(), repoPath, bucketName)
 }
 
+// SetCacheLimits sets to an existing one if possible.
+func (kpl *KopiaPersisterLight) SetCacheLimits(repoPath string, cacheOpts *content.CachingOptions) error {
+	bucketName := os.Getenv(S3BucketNameEnvKey)
+	err := kpl.kc.ConnectRepo(context.Background(), repoPath, bucketName)
```

Comment:
It looks like we're connecting to the repository three times in a row. Once in `getPersister`, then again here, then immediately after in `kc.SetCacheLimits`. Was that intended?

Reply:
good point.  I removed two redundant connects.